File: comunidata-api/src/main/java/com/senasoft/comunidataapi/csv/dto/response/CsvUploadResponseDTO.java
package com.senasoft.comunidataapi.csv.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/** DTO de respuesta para la carga de CSV. */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CsvUploadResponseDTO {
    private String message;
    private Integer totalRecords;
    private Integer normalizedRecords;
    private Integer recordsWithErros;
    private String batchId;
    private String processingStatus;
}

File: comunidata-ui/lib/dto/chat/response/csv_upload_response_dto.dart
class CsvUploadResponseDTO {
  final String message;
  final int totalRecords;
  final int normalizedRecords;
  final int recordsWithErros;
  final String batchId;
  final String processingStatus;

  CsvUploadResponseDTO({required this.message, required this.totalRecords, required this.normalizedRecords,
    required this.recordsWithErros, required this.batchId, required this.processingStatus});
}